### PR TITLE
add dps document

### PR DESCRIPTION
**Pull request by ihere1**

### CODE REVIEW

File: docs/iot-devkit/devkit_dps.md [hunk 1]
```diff
@@ -0,0 +1,149 @@
...
+
+	![IoT Workbench: Device - > Upload](media/iot-workbench-device-upload.png)
+
+4. VS Code then starts verifying and uploading the code to your DevKit.
+
+	![IoT Workbench: Device - > Uploaded](media/devkit-dps/device-uploaded.png)
+
+5. The DevKit reboots and starts running the code.
+
+6. Note down **DevKit MAC Address** and **DevKit Firmware Version** value from serial monitor.
```
Comment (lirenhe) [inline docs/iot-devkit/devkit_dps.md]: How to open serial monitor in VS code? need to provide some guide

File: docs/iot-devkit/devkit_dps.md [hunk 2]
```diff
@@ -0,0 +1,149 @@
...
+In this tutorial, you learn:
+- How to set up the Device Provisioning Service configuration on the device.
+- How to save Unique Device Secret on STSAFE security chip.
+- How to generate X.509 certificate.
+- How to create a device enrollment entry in the Device Provisioning Service.
+
+## What you need
+
+Finish the [Getting Started Guide](./devkit-get-started.md) to:
+
```
Comment (lirenhe) [inline docs/iot-devkit/devkit_dps.md]: better organize this part.

File: docs/iot-devkit/devkit_dps.md [hunk 3]
```diff
@@ -0,0 +1,149 @@
...
+* Activate a [free 30-day trial Microsoft Azure account](https://azure.microsoft.com/free/).
+* Claim your [Azure credit](https://azure.microsoft.com/pricing/member-offers/msdn-benefits-details/) if you are an MSDN or Visual Studio subscriber.
+
+Create and link IoT Hub with Device Provisioning Service instance with [Set up auto provisioning](https://docs.microsoft.com/en-us/azure/iot-dps/quick-setup-auto-provision).
+
+## Open the project folder
+
+### Start VS Code
+
+* Make sure your DevKit is not connected to your PC.
```
Comment (ArthurMa1978) [inline docs/iot-devkit/devkit_dps.md]: please refer other doc to update this section.

Comment (ihere1) [reply docs/iot-devkit/devkit_dps.md]: Nearly every doc but get-started have this part. It is a little wired to jump from a mini solution tutorial to another. Which doc should I refer to.

Comment (ihere1) [reply docs/iot-devkit/devkit_dps.md]: If merge of this part is needed. We can open a new pr.

**Pull request merged**